tracer/test/Datadog.Trace.Tests/Configuration/IntegrationSettingsTests.cs

```diff
@@ -27,6 +30,21 @@ public void IntegrationEnabled(string settingName, string settingValue, bool exp
             Assert.Equal(expected, settings.Enabled);
         }
 
+        [Theory]
+        [InlineData("DD_TRACE_MYSQL_ENABLED", "false", false)]
+        [InlineData("DD_TRACE_MySql_ENABLED", "false", false)]
```

Comment:
Would you mind adding 2 more lines for `"true", true` just to cover all the bases?

```suggestion
        [InlineData("DD_TRACE_MYSQL_ENABLED", "false", false)]
        [InlineData("DD_TRACE_MySql_ENABLED", "false", false)]
        [InlineData("DD_TRACE_MYSQL_ENABLED", "true", true)]
        [InlineData("DD_TRACE_MySql_ENABLED", "true", true)]
```